### PR TITLE
Enable the sinker alerts

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow-up of https://github.com/openshift/release/pull/4881/

### CODE REVIEW

**Comment by hongkailiu** [comment]
Expecting failure of `ci/prow/generated-dashboards`

**Comment by hongkailiu** [comment]
`ci/prow/generated-dashboards` should be fixed.

**Comment by hongkailiu** [comment]
Also checked.
The current one does more than the existing `absent(up{job="sinker"} == 1)`.

File: cluster/ci/monitoring/mixins/prometheus_out/prometheus_rule_prow.yaml [hunk 1]
```diff
@@ -260,3 +260,23 @@ spec:
...
+    rules:
+    - alert: SinkerNotRemovingPods
+      annotations:
+        message: Sinker has not removed any Pods in the last hour, likely indicating
+          an outage in the service.
+      expr: |
+        absent(sum(rate(sinker_pods_removed[1h]))) == 1
+      for: 5m
+      labels:
+        severity: critical
```
Comment (petr-muller) [inline cluster/ci/monitoring/mixins/prometheus_out/prometheus_rule_prow.yaml]: `critical` does not sound right: last time this happened everything worked for few weeks until we noticed we're using quite a lot of memory, but caused no real problems...

Comment (hongkailiu) [reply cluster/ci/monitoring/mixins/prometheus_out/prometheus_rule_prow.yaml]: Good catch.
Updated also the other places where we use `critical`.

We have not used this label `severity` yet and the meaning of the label follows the examples in the [prometheus-doc](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/#templating).

I think the `severity` here is the method to notify the Ops, here `slack` and `page` in the above doc.

**Comment by hongkailiu** [comment]
Checked on the UI: https://prometheus-prow-monitoring.svc.ci.openshift.org/alerts
The new alerts are added.

**Pull request merged**